Scripts/Python/xCensorFilters.py

```diff
@@ -369,14 +367,8 @@
             "jizz"          : Rating(xRatedX),\
             "kawk"          : Rating(xRatedX),\
             "kike"          : Rating(xRatedX),\
-            "klootzak"          : Rating(xRatedR),\
             "koon"          : Rating(xRatedX),\
-            "knulle"          : Rating(xRatedX),\
-            "kuk"          : Rating(xRatedR),\
-            "kuksuger"          : Rating(xRatedX),\
             "kunt"          : Rating(xRatedX),\
-            "kurac"          : Rating(xRatedR),\
-            "kurwa"          : Rating(xRatedX),\
             "kyrpa"          : Rating(xRatedR),\
```

Comment:
You missed the Finnish penis...es. https://en.wiktionary.org/wiki/kyrp%C3%A4

Reply:
I was wondering what that meant 🤣